src/plots/polar/polar.js

```diff
@@ -483,9 +483,12 @@ proto.updateRadialAxisTitle = function(fullLayout, polarLayout, _angle) {
     var sina = Math.sin(angleRad);
 
     var pad = 0;
+
+    // Hint: no need to check if there is in fact a title.text set
+    // because if plot is editable, pad needs to be calculated anyways.
```

Comment:
Hmm. Could we elaborate here? Why does `pad` need to be calculated?

Reply:
Thanks. Done.